Gateway/Config/Config.php

```diff
@@ -321,7 +314,7 @@ public function getPaymentPlansConfig(): PaymentPlansConfigInterface
             try {
                 $plansConfig->updateFromApi();
             } catch (RequestError $e) {
-                // TODO: log error (circumvent circular dependency between Logger & Config)
+                $this->logger->error('getPaymentPlansConfig Error : ',[]);
```

Comment:
maybe add at least `$e->getMessage()` or `$e->getTraceAsString()` instead of empty `[]` ?

Reply:
done